reference/7.2/Microsoft.PowerShell.Core/About/about_PowerShell_Config.md

```diff
@@ -39,358 +72,380 @@ installation.
+found in the registry have precedence. Next PowerShell reads the JSON
+configuration. Any settings found under `PowerShellPolicies` as long as they
+are not found in the registry. Finally, any remaining settings found at the
+root level of the JSON configuration are used.
 
-The names of the experimental features to enable in PowerShell. By default, no
-experimental features are enabled. The default value is an empty array.
+For more information, see [about_Group_Policy_Settings][07].
 
-```Schema
-"ExperimentalFeatures": ["<experimental-feature-name>", ...]
-```
+## Settings for non-Windows platforms
 
-Where:
+The following settings only apply to Linux and macOS platforms.
 
-- `<experimental-feature-name>` is the name of an experimental feature to
-  enable.
+The following keys are used to configure PowerShell's logging for Linux and
+macOS.
 
-The following example enables the **PSImplicitRemoting** and
-**PSUseAbbreviationExpansion** experimental features when PowerShell starts up.
+- `LogChannels`
+- `LogIdentity`
+- `LogKeywords`
+- `LogLevel`
 
-```json
-{
-  "ExperimentalFeatures": [
-    "PSImplicitRemotingBatching",
-    "PSUseAbbreviationExpansion"
-  ]
-}
-```
+For a full description of PowerShell logging for non-Windows systems, see
+[about_Logging_Non-Windows][02].
 
-For more information on experimental features, see
-[Using experimental features](/powershell/scripting/learn/experimental-features).
+## Common configuration settings
 
-## Non-Windows logging configuration
+The following settings are available on all supported platforms.
 
-> [!IMPORTANT]
-> The configuration options in this section only apply to macOS and Linux.
-> Logging for Windows is managed by the Windows Event Viewer.
+- `ConsoleSessionConfiguration`
+- `ExperimentalFeatures`
+- `ModuleLogging`
+- `ProtectedEventLogging`
+- `PSModulePath`
+- `ScriptBlockLogging`
+- `ScriptExecution`
+- `Transcription`
+- `UpdatableHelp`
 
-PowerShell's logging on macOS and Linux can be configured in the PowerShell
-configuration file. For a full description of PowerShell logging for
-non-Windows systems, see [About Logging](./about_Logging_Non-Windows.md).
+### ConsoleSessionConfiguration
 
-### LogIdentity
+This key contains two subkeys:
 
-> [!IMPORTANT]
-> This setting can only be used in macOS and Linux.
+- `EnableConsoleSessionConfiguration` - to enable session configurations, set
+  the value to `true`.
 
-Sets the identity name used to write to the system log. The default value is
-"powershell".
+- `ConsoleSessionConfigurationName` - Specifies the name of configuration
+  endpoint in which PowerShell is run. This can be any endpoint registered on
+  the local machine including the default PowerShell remoting endpoints or a
+  custom endpoint having specific user role capabilities.
 
-```Schema
-"LogIdentity": "<log-identity>"
+```json
+{
+  "ConsoleSessionConfiguration": {
+    "EnableConsoleSessionConfiguration": false,
+    "ConsoleSessionConfigurationName" : []
```

Comment:
Is this value required when enable is set to `false`?

What's the behavior when enable is set to `true` and this value is an empty array?